Some changes

eighty88 opened the pull request:
Sorry, I use translation web site,

# Smooth Font Rendering
## prism.lcdtext
Changed "prism.lcdtext" to false.

Before:
![Screenshot](https://user-images.githubusercontent.com/49605393/148902665-f72390d8-c09a-4a15-b0ce-953c27faec30.png)
After:
![Screenshot](https://user-images.githubusercontent.com/49605393/148902697-a014195d-bd5f-4a72-ac06-6ac0f1a11793.png)

## Noto Sans
Added "Noto Sans" that supports many languages so that the characters do not collapse when displayed in multiple languages.

In japanese,

Before:
![Screenshot](https://user-images.githubusercontent.com/49605393/148903036-d85a51ab-ab80-47da-8cf0-0cba5f3a2f91.png)
After:
![Screenshot](https://user-images.githubusercontent.com/49605393/148903053-be24d51f-62db-4951-aa26-d33a774e3872.png)

### Noto Sans license...

A is SIL Open Font License 1.1.
I wrote the license name in a place where people can see it because it will be treated as built-in.
![Screenshot](https://user-images.githubusercontent.com/49605393/148903575-f5b14bed-9a64-46ff-9f47-9768c1e34b9c.png)
And I put the License text together with the font file.

# Japanese localization of GUI

Updated GUI_ja_JP.properties file.

Before:
![Screenshot](https://user-images.githubusercontent.com/49605393/148905652-f7b51877-078f-4220-bbaa-62308225c3ca.png)

After:
![Screenshot](https://user-images.githubusercontent.com/49605393/148905687-c2f2ecc1-8701-484b-b530-3a2bc1655c70.png)

eighty88 committed:
[Add] Add Noto Sans for multilingual display.

eighty88 committed:
[Remove] Remove Reboto Regular because it hasn't used.

eighty88 committed:
[Update] Update Fonts
[Remove] Roboto Regular
[Add] Noto Sans

eighty88 committed:
[Update] Update Fonts
[Remove] Roboto Regular
[Add] Noto Sans

eighty88 committed:
[Add] Add window "Software and License"

eighty88 committed:
[Update] Set false "prism.lcdtext" for smooth font rendering

eighty88 committed:
[Update] Add Noto Sans Fonts (Japan, China)

eighty88 committed:
[Update] Add Noto Sans Fonts (Japan, China)

eighty88 committed:
[Update] Add Noto Sans Fonts (Japan, China)

eighty88 committed:
[Remove] Remove Noto Sans Fonts

eighty88 committed:
[Add] Add Noto Sans CJK Regular

eighty88 committed:
[Update] Update Fonts

eighty88 committed:
[Update] Update Japanese Language

eighty88 committed:
[Add] Add NotoSansCJK-Regular.otf.License.txt

OfficialDonut commented:
- Setting "prism.lcdtext" made it look worse if anything imo.
- According to the [JavaFX CSS Reference Guide](https://openjfx.io/javadoc/17/javafx.graphics/javafx/scene/doc-files/cssref.html#introlimitations): `7. JavaFX CSS does not support comma-separated series of font family names in the -fx-font-family property.`
- The license menu is overkill, a text file is sufficient

eighty88 committed:
[Update] Set true "prism.lcdtext"

eighty88 committed:
[Update] Make only one font family name in "-fx-font-family".

eighty88 committed:
[Update] Remove SoftwareLicenseGui.java

eighty88 committed:
[Remove] Remove item of Licenses from lang files.

eighty88 commented:
### Set true "prism.lcdtext"

The color of the text is still wrong in my environment, so I will make it false only when I use it myself.

### Make only one font family name in "-fx-font-family".

I wrote it as the IDE complemented it.
fixed.

### Remove License Menu

I was looking at the license for the abridged version, but from what I've seen of the original, it looks like I don't need to go that far!
So I deleted it.
